src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java

```diff
@@ -44,5 +45,12 @@ public String finalPage(ModelMap model) {
 		
 		return FINAL_PAGE_KEY;
 	}
+	
+	@RequestMapping(value = "/Guest", method = RequestMethod.GET)
```

Comment:
it doesnt same with the task, it should be /{some input} and then show error page, not /Guest and then show error page